x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -172,8 +174,14 @@ export const useMatrixHistogram = ({
   }, [indexNames, endDate, filterQuery, startDate, stackByField, histogramType, threshold]);
 
   useEffect(() => {
+    if (!skip) {
```

Comment:
Thanks to @XavierM and @patrykkopycinski that helped me figure out best way to give myself access to a callback I could use to invoke search on invocation, not on prop change.